reference/5.1/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -356,15 +359,14 @@ You can pipe any object to `Out-File`.
 
 ## NOTES
 
-The `Out` cmdlets do not format objects; they just render them and send them to the specified
-display destination. If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a
-formatting cmdlet before rendering it.
+If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a formatting cmdlet
+before rendering it.
 
 To send a PowerShell command's output to the `Out-File` cmdlet, use the pipeline. You can store data
 in a variable and use the **InputObject** parameter to pass data to the `Out-File` cmdlet.
 
-`Out-File` sends data but it does not produce any output objects. If you pipe the output of
-`Out-File` to `Get-Member`, the `Get-Member` cmdlet reports that no objects were specified.
+`Out-File` sends data but it does not produce any output objects to the pipeline. It only writes
```

Comment:
Perhaps more simply: `Out-File` saves data to a file, but it does not produce any output to the pipeline.

Reply:
I think this does make it simpler. Fixed